### DISCUSSION TITLE
Merged into #6834: Visual Editor save UX

### DISCUSSION THREAD

**Discussion by seanwbarry** [growthbook/growthbook#2762] [Ideas]
This thread was merged into #6834 (What should change in the Visual Editor and redirects?). Please continue there.

Original converted issue title: Request - Visual Editor UX Improvement.

**Comment by alyssamichelle** [comment]
Continuing in #6834 (What should change in the Visual Editor and redirects?). The Save-button UX is part of the Visual Editor / redirects question.
